about.html

```diff
@@ -0,0 +1,98 @@
+            <h2 class="main-header-text-about">Kano state <br> tech summit 2015</h2>
+        </div>
+        <div class="about-description">
+            <h3 class="about-description-text">
+                The Global Summit brings together the community of  
+                experts, academics, and activists who comprise the Creative Commons affiliate network in a different country every two years.
+                This year, we hope to expand our invitation list including organizations and individuals.
+            </h3>
+            <h3 class="about-description-text-desktop">The Global Summit brings together the community of experts, academics, and activists who comprise the Creative Commons affiliate network in a different country every two years.
+                This year, we hope to expand our invitation list including organizations and individuals who want to work with us on shared projects that advance the cause of the Commons, free culture and open knowledge.
+                So if you're active and engaged in the worlds of open content and knowledge - free software advocates, wikipedians, Open knowledge, galleries, libraries, museums, archives, governments and foundations, lawyers, and activists - we hope you'll consider joining us this year to build a stronger, more vibrant commons together</h3>
+        </div>
+        <div class="about-contact">
+            <h3 class="first-line">Please contact us per Email for nay further questions about Kano State Tech Summit 2015!</h3>
+            <h3 class="second-line">techsummit2015@kano.org</h3>
+        </div>
+    </div>
+    <ul class="creative-common-section">
+        <li class="creative-common-text-one">Kano State <br> Tech Summit 2015 Logo</li>
+        <li class="creative-common-text-one-desktop">Kano State Tech Summit 2015 Logo</li>
+        <i class="fa-solid fa-horizontal-rule line-red-about"></i>
+        <li class="creative-common-text-two">The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7, July.</li>
+        <li class="creative-common-text-two-desktop">The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7, July.
+            After online public voting and Selection Committee’s judging process, the logo from Naresh Agrawal won the prize.
+            After consultation with Mr. Agrawal, we optimized the prize-winning-logo for this year’s summit as follows:</li>
+        <li><img src="./img/global-summit-logo.png" alt="" class="global-summit-logo"></li>
+    </ul>
+    <div class="see-the-past">
+        <ul class="see-the-past-items">
+            <li class="see-the-past-text-one">See the past Kano State Tech Summits</li>
+            <i class="fa-solid fa-horizontal-rule line-red-about"></i>
+            <li class="see-the-past-text-two">Take a look at the past two Kano State Tech Summits which took place in Dawanau and in Hadejia</li>
+        </ul>
+    </div>
+    <div class="past-images">
```

Comment:
- [ ] It would be better if you wrap this section with a `section` tag for semantic reasons :+1: